### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by kyam-mayden**
All responsiveness issues resolved

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -2,57 +2,65 @@
 	box-sizing:border-box;
 }
 
+body{
+	min-width:550px;
```
Comment (nsantiagoblair) [inline styles.css:6]: smells like code smell!

File: styles.css [hunk 2]
```diff
@@ -63,14 +71,33 @@
...
+	width:30%;
+	height:100px;
+	background-color:gray;
+	margin:0 auto;
+	position:relative;
+	opacity:0.7;
+}
+
+.buttonHolder div h2{
+	margin:0 auto;
```
Comment (nsantiagoblair) [inline styles.css]: is `margin: 0 auto` working? You can try `text-align: center` in the parent element

File: styles.css [hunk 3]
```diff
@@ -92,11 +119,9 @@ a[href="index.html"]{
 
 .social a{
 	height:30px;
-	background-repeat: no-repeat;
-	background-position: center;
+	background-repeat:no-repeat;
```
Comment (nsantiagoblair) [inline styles.css:122]: what background image do you want to `no-repeat`?

**Review by nsantiagoblair — COMMENTED** [review]
AWESOME!! 😸

File: styles.css (hunk 1 above)
Comment (kyam-mayden) [reply styles.css:6]: This was put in to limit the viewport size...as I didn't want to find solutions for a smaller screen size.

File: styles.css (hunk 2 above)
Comment (kyam-mayden) [reply styles.css]: margin was doing nothing & has been removed

File: styles.css (hunk 3 above)
Comment (kyam-mayden) [reply styles.css:122]: This line is causing the image to display once only

**Review by nsantiagoblair — COMMENTED** [review]
Looks good 🎉

**Pull request merged**